test/runAll.spec.js

```diff
@@ -11,6 +12,9 @@ sgfMock.mockImplementation((params, callback) => {
 const scripts = { mytask: 'echo "Running task"' }
 
 describe('runAll', () => {
+  beforeEach(() => {
+    global.console = makeConsoleMock()
```

Comment:
I think this mock should be cleared after tests are run.

Reply:
Also I know that this method sometimes fails. See https://github.com/okonet/lint-staged/pull/304/files#diff-faa503f61f8addeb603095cc2f40c82eR17

@cameronhunter do you have any insights on this?

Reply:
Clobbering the global console in tests isn't a good idea, anything written to the console by another file/package will also be captured. Instead prefer to inject a logger into your package allowing you to limit what is captured, the injected logger can be defaulted to `console` for a simpler non-test usage.